marconi-sidechain/test/Spec/Marconi/Sidechain/Api/Routes/Golden/address-utxo-response.json

```diff
@@ -12,8 +12,10 @@
     {
         "address": "addr_test1vpfwv0ezc5g8a4mkku8hhy3y3vp92t7s3ul8g778g5yegsgalc6gc",
         "blockHeaderHash": "6161616161616161616161616161616161616161616161616161616161616161",
-        "datum": "34",
-        "datumHash": "eb8649214997574e20c464388a172420d25403682bbbb80c496831c8cc1f8f0d",
+        "datum": {
```

Comment:
This change was intentional? (Just checking)

Reply:
yes, they want a standardised json of the datum, while we were returning the CBOR before.